Resources/ProxyProvider/test/Test_DIRACCAProxyProvider.py

```diff
@@ -0,0 +1,177 @@
+""" This is a test of the DIRACCAProxyProvider
+"""
+
+# pylint: disable=invalid-name,wrong-import-position,protected-access
+import os
+import re
+import sys
+import shutil
+import commands
+import unittest
+import tempfile
+
+from DIRAC.Core.Base.Script import parseCommandLine
+parseCommandLine(ignoreErrors=True)
```

Comment:
If you do `parseCommandLine`, then it is an integration test, that should be in `DIRAC/tests`. This directory is for unit tests

Reply:
Yes, you right. It's unit tests, so this lines not needed. Removed.